src/common/messages.cpp

```diff
@@ -116,6 +116,10 @@ bilingual_str PSBTErrorString(PSBTError err)
             return Untranslated("External signer failed to sign");
         case PSBTError::UNSUPPORTED:
             return Untranslated("Signer does not support PSBT");
+        case PSBTError::INCOMPLETE:
+            return Untranslated("Input needs additional signatures or other data");
```

Comment:
It looks like there is no test coverage for this yet. Is there coverage for this in a follow-up PR (maybe Musig2) or are you writing a test? Otherwise I can give it a shot.

Reply:
I don't think there's any specific coverage for this, feel free to tackle it.

Reply:
Follow up: I created a PR to add test coverage: https://github.com/bitcoin/bitcoin/pull/33025
Will ensure it passes all the CI checks, 🙇